models/forum.js

```diff
@@ -16,7 +16,7 @@ var ForumSchema = new Schema({
 	},
 	created: {
 		type: Date,
-		default: Date.now
+		default: new Date()
```

Comment:
Any particular reason why you introduce `new Date()` over the default Mongoose method `Date.now`?

Reply:
I thought both new Date() and Date.now are javascript methods and not Mongoose. I changed it to new Date() because the format is more readable.